Author: Adam Di Carlo <aph@debian.org>.
Description: * docbook-4.{2,3}/catalog.xml: Adds system identifiers to the catalog.
* docbook-4.x/docbook.cat: Likewise.

<URL:http://bugs.debian.org/301157>
<URL:http://bugs.debian.org/304057>
<URL:http://packages.debian.org/changelogs/pool/main/d/docbook-xml/current/changelog#versionversion4.2-8>
<URL:http://packages.debian.org/changelogs/pool/main/d/docbook-xml/current/changelog#versionversion4.4-1>
<URL:http://packages.debian.org/changelogs/pool/main/d/docbook-xml/current/changelog#versionversion4.5-2>
--- a/docbook-4.1.2/docbook.cat
+++ b/docbook-4.1.2/docbook.cat
@@ -23,6 +23,8 @@
   -- DocBook driver file .................................................. --
 
 PUBLIC "-//OASIS//DTD DocBook XML V4.1.2//EN" "docbookx.dtd"
+SYSTEM "http://www.oasis-open.org/docbook/xml/4.1.2/docbookx.dtd" "docbookx.dtd"
+SYSTEM "http://docbook.org/xml/4.1.2/docbookx.dtd" "docbookx.dtd"
 DTDDECL "-//OASIS//DTD DocBook XML V4.1.2//EN" "/usr/share/xml/declaration/xml.dcl"
 
   -- ...................................................................... --
--- a/docbook-4.2/catalog.xml
+++ b/docbook-4.2/catalog.xml
@@ -28,6 +28,12 @@
 <public publicId="-//OASIS//DTD DocBook XML V4.2//EN"
         uri="docbookx.dtd"/>
 
+<system systemId="http://www.oasis-open.org/docbook/xml/4.2/docbookx.dtd"
+        uri="docbookx.dtd"/>
+
+<system systemId="http://docbook.org/xml/4.2/docbookx.dtd"
+        uri="docbookx.dtd"/>
+
 <!-- ...................................................................... -->
 <!-- DocBook modules ...................................................... -->
 
--- a/docbook-4.2/docbook.cat
+++ b/docbook-4.2/docbook.cat
@@ -24,6 +24,10 @@
 
 PUBLIC "-//OASIS//DTD DocBook XML V4.2//EN"
        "docbookx.dtd"
+SYSTEM "http://www.oasis-open.org/docbook/xml/4.2/docbookx.dtd"
+       "docbookx.dtd"
+SYSTEM "http://docbook.org/xml/4.2/docbookx.dtd"
+       "docbookx.dtd"
 DTDDECL "-//OASIS//DTD DocBook XML V4.2//EN"
         "/usr/share/xml/declaration/xml.dcl"
 
--- a/docbook-4.3/catalog.xml
+++ b/docbook-4.3/catalog.xml
@@ -28,6 +28,12 @@
 <public publicId="-//OASIS//DTD DocBook XML V4.3//EN"
         uri="docbookx.dtd"/>
 
+<system systemId="http://www.oasis-open.org/docbook/xml/4.3/docbookx.dtd"
+        uri="docbookx.dtd"/>
+
+<system systemId="http://docbook.org/xml/4.3/docbookx.dtd"
+        uri="docbookx.dtd"/>
+
 <!-- ...................................................................... -->
 <!-- DocBook modules ...................................................... -->
 
--- a/docbook-4.3/docbook.cat
+++ b/docbook-4.3/docbook.cat
@@ -24,6 +24,10 @@
 
 PUBLIC "-//OASIS//DTD DocBook XML V4.3//EN"
        "docbookx.dtd"
+SYSTEM "http://www.oasis-open.org/docbook/xml/4.3/docbookx.dtd"
+       "docbookx.dtd"
+SYSTEM "http://docbook.org/xml/4.3/docbookx.dtd"
+       "docbookx.dtd"
 DTDDECL "-//OASIS//DTD DocBook XML V4.3//EN"
         "/usr/share/xml/declaration/xml.dcl"
 
--- a/docbook-4.4/docbook.cat
+++ b/docbook-4.4/docbook.cat
@@ -24,6 +24,10 @@
 
 PUBLIC "-//OASIS//DTD DocBook XML V4.4//EN"
        "docbookx.dtd"
+SYSTEM "http://www.oasis-open.org/docbook/xml/4.4/docbookx.dtd"
+       "docbookx.dtd"
+SYSTEM "http://docbook.org/xml/4.4/docbookx.dtd"
+       "docbookx.dtd"
 DTDDECL "-//OASIS//DTD DocBook XML V4.4//EN"
         "/usr/share/xml/declaration/xml.dcl"
 
--- a/docbook-4.5/docbook.cat
+++ b/docbook-4.5/docbook.cat
@@ -24,6 +24,10 @@
 
 PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
        "docbookx.dtd"
+SYSTEM "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd"
+       "docbookx.dtd"
+SYSTEM "http://docbook.org/xml/4.5/docbookx.dtd"
+       "docbookx.dtd"
 DTDDECL "-//OASIS//DTD DocBook XML V4.5//EN"
         "/usr/share/xml/declaration/xml.dcl"
 
